format/spec.md

```diff
@@ -158,27 +158,27 @@ Readers should be more permissive because v1 metadata files are allowed in v2 ta
 
 Readers may be more strict for metadata JSON files because the JSON files are not reused and will always match the table version. Required fields that were not present in or were optional in prior versions may be handled as required fields. For example, a v2 table that is missing `last-sequence-number` can throw an exception.
 
-### Writing data files
+#### Writing data files
 
 All columns must be written to data files even if they introduce redundancy with metadata stored in manifest files (e.g. columns with identity partition transforms). Writing all columns provides a backup in case of corruption or bugs in the metadata layer.
 
 Writers are not allowed to commit files with a partition spec that contains a field with an unknown transform.
 
-## Schemas and Data Types
+### Schemas and Data Types
```

Comment:
Because the container (specification) is going to be H2, I think this is a decision point. Do we care that things beyond this point are nested within Specification? I think I would rather have headings that are more distinct in the body of the spec than worry about these sections being nested within `Spec`. I'm curious what other people think though. @RussellSpitzer?

Reply:
ok, I just changed spec to H2 and reverted other changes. Rendered site LGTM. 

Updated the PR.

Reply:
@rdblue I think we want to add additional nesting because we can limit the depth in the ToC configuration.  That way we can have proper structure and not have the table of contents explode.

Reply:
The maximum depth we have is 3, do we have any plan to add depth limit? If not, maybe we don't have to argue on this.

Reply:
I really don't have any preference. I'm fine with just pumping all the nesting one more step deep.

Reply:
This isn't right though.  The Specification header should be above the others (e.g. `Schemas and Data Types`) not at the same level.

Reply:
Okay, that sounds fine to me.